internal/monocular/client.go

```diff
@@ -44,12 +45,10 @@ func New(u string) (*Client, error) {
 
 	return &Client{
 		BaseURL: u,
-		Log:     nopLogger,
+		Log:     slog.New(slog.NewTextHandler(io.Discard, nil)),
```

Comment:
I think we want to remove this logger that is passed around and address logging directly from slog.